localstack/services/apigateway/models.py

```diff
@@ -68,6 +69,8 @@ class ApiGatewayStore(BaseStore):
     # maps resource ARN to tags
     TAGS: Dict[str, Dict[str, str]] = CrossRegionAttribute(default=dict)
 
+    domain_names: Dict[str, DomainName] = LocalAttribute(default=dict)
```

Comment:
tiny nit: can we maybe group it visually to move the `LocalAttribute`s together, and have the `CrossRegionAttribute` below it? Would also be great to add a short comment (or docstring) to the variable, to be in line with the other fields in this class 👍